_data-prepper/pipelines/configuration/buffers/bounded-blocking.md

```diff
@@ -14,8 +14,8 @@ The default buffer. Memory-based.
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
-buffer_size | No | Integer | The maximum number of records the buffer accepts. Default is `512`.
-batch_size | No | Integer | The maximum number of records the buffer drains after each read. Default is `8`.
+buffer_size | No | Integer | The maximum number of records the buffer accepts. Default value is `512`.
```

Comment:
I think the default buffer and batch size was changed recently (https://github.com/opensearch-project/documentation-website/pull/2574)